pkg/logging/log_viewer.go

```diff
@@ -98,8 +103,8 @@ func (lvo *LogViewOptions) Validate() error {
 // Implements functionality for loading the logging configuration and
 // fetching/outputting container logs based on its internal LogViewOptions.
 type ContainerLogViewer struct {
-	// Logging configuration.
-	loggingConfig LogConfig
+	// Logging logDriver.
+	logDriver string
```

Comment:
why making it a specific field ?, we may add other configuration in the future different from driver

Reply:
has been revert